src/main/java/org/opentripplanner/routing/graph/Graph.java

```diff
@@ -249,7 +250,7 @@
     public final BiMap<Trip,Trip> interlinedTrips = HashBiMap.create();
 
     /** Pre-generated transfers between all stops. */
-    public final Multimap<StopLocation, SimpleTransfer> transfersByStop = HashMultimap.create();
+    public final Multimap<StopLocation, SimpleTransfer> transfersByStop = Multimaps.synchronizedMultimap(HashMultimap.create());
```

Comment:
`Graph#transfersByStop` is only updated by the `DirectTransferGenerator` -- it was modified to be synchronized so that transfers could be generated in parallel.

I've modified the code so that this implementation detail is hidden in `DirectTransferGenerator`.